integrations/redis.go

```diff
@@ -0,0 +1,125 @@
+package integrations
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/go-redis/redis/v7"
```

Comment:
style nit: we usually put the imports in 3 groups: stdlibs, libraries from the same git repo, and everything else. so add a blank line after this line to separate the two groups.